library/cisco_imc_virtual_drive.py

```diff
@@ -186,8 +198,10 @@ def virtual_drive(server, module):
                 results["changed"] = not _exists
                 return results, False
 
+            print "size is %s" % ansible["size"]
```

Comment:
@dsoper2 ansible frowns upon any print statements. we should probably remove this line.